src/components/AppSettingsMenu.vue

```diff
@@ -267,9 +267,16 @@ p.app-settings {
 		line-height: normal;
 		min-height: 44px;
 		font-size: unset;
+		opacity: .7;
 	}
 }
 .material-design-icon.lock-icon {
 	margin-right: 10px;
 }
+.material-design-icon {
+	opacity: .7;
+}
```

Comment:
```suggestion
```

Doesn't do anything and is not required.

Reply:
left over sorry